### PR TITLE
Add new statistics introduced in 1.31.0 to gLiveView.sh

### PR DESCRIPTION
**Pull request by orpheus-antpool**
Cardano Node v.1.31.0 adds several new statistics to Prometheus/EKG:
- number of forks
- delay of receiving last block since reference tip
- percentage of blocks received within 3s since node start
- number of late blocks since node start

I've added these statistics to gLiveView.sh to provide better health monitoring of the nodes. They help the community identify block propagation issues and mitigate it by tweaking the topology and/or connectivity of the nodes. To save space, I removed Tip (node) which is redundant given Tip ref and Tip diff.

### CODE REVIEW

**Review by rdlrt — COMMENTED** [review]
Thanks for the PR!
One of the pre-req to this would be changing minimum baseline from 1.30.1 to 1.31.0 in env file

**Comment by Scitz0** [comment]
Thanks for PR, but shouldn't we add cdfOne and cdfFive as well and not just cdfThree?

**Comment by orpheus-antpool** [comment]
We could, I was just worried about eating up too much space, which was a concern last time we discussed about additional metrics :) I can easily add another line in the block propagation section and show cdfOne, cfdThree, cfdFive there. Instead of cdfThree on the fist line, I can add  % of blocks served (served/total since node start), which gives an idea about how useful the node is to others (i.e. do they find out about new blocks from me or from somebody else).

**Comment by Scitz0** [comment]
> We could, I was just worried about eating up too much space, which was a concern last time we discussed about additional metrics :) I can easily add another line in the block propagation section and show cdfOne, cfdThree, cfdFive there. Instead of cdfThree on the fist line, I can add  % of blocks served (served/total since node start), which gives an idea about how useful the node is to others (i.e. do they find out about new blocks from me or from somebody else). Any thoughts on this?

Sounds good, give it a try and I'll test when pushed.

File: scripts/cnode-helper-scripts/gLiveView.sh [hunk 1]
```diff
@@ -56,7 +56,7 @@ setTheme() {
 # Do NOT modify code below           #
 ######################################
 
-GLV_VERSION=v1.23.1
+GLV_VERSION=v1.23.2
```
Comment (Scitz0) [inline scripts/cnode-helper-scripts/gLiveView.sh:59]: Looks great, only thing as this is a bigger change and not a fix/patch, would be to bump version to 1.24.0 instead.

**Review by Scitz0 — APPROVED** [review]
👍

File: scripts/cnode-helper-scripts/gLiveView.sh (hunk 1 above)
Comment (orpheus-antpool) [reply scripts/cnode-helper-scripts/gLiveView.sh]: Sounds good, looking forward to the release.

**Pull request merged**